s31321-JakubTrzaskowski/Powershell/powershell_Zadania.txt

```diff
@@ -0,0 +1,280 @@
+*PS C:\Users\Lenovo Yofa> Write-Host $env:TEMP\ListaPolecen
+C:\Users\Lenovo Yofa\AppData\Local\Temp\ListaPolecen
+Wynik zostal zapisany w pliku.
+9.A teraz bysmy chcieli nic z tym nie robic tylko by sie przetworzylo ale nie wypisywalo.
+*PS C:\Users\Lenovo Yofa> Get-Module -ListAvailable | ? {$_.Version -ge "2.0.0.0"} | % {Get-Command -Module $_.Name} | ? {($_.Name -like "Add-*") -or ($_.Name -like "Get-*")} | Out-Null
+Polecenie wykonalo sie, czyli zostaly przefiltrowane polecenie modulow wedlug zadanych parametrow, ale nic nie zostalo wyswietlone ani zapisane, tylko i wylacznie dane zostaly przetworzone.
+10.Dobrze, to listy już mamy opanowane. To teraz w końcu byśmy chcieli załadować jakiś moduł np. BitsTransfer (do zarządzania przesyłaniem danych w tle).
+*PS C:\Users\Lenovo Yofa>  Import-Module BitsTransfer
+lub przy pomocy aliasu
+*PS C:\Users\Lenovo Yofa> impo BitsTransfer
+Zaladowany zostal modul do zarzadzania przesylaniem danych w tle.
+11.Wylistujmy wszystkie komendy w tym module.
+PS C:\Users\Lenovo Yofa> Get-Command -Module BitsTransfer
+CommandType     Name                                               Version    Source
+-----------     ----                                               -------    ------
+Cmdlet          Add-BitsFile                                       2.0.0.0    BitsTransfer
+Cmdlet          Complete-BitsTransfer                              2.0.0.0    BitsTransfer
+Cmdlet          Get-BitsTransfer                                   2.0.0.0    BitsTransfer                               2.0.0.0    BitsTransfer
+...
+Wylistowane zostale wszystkie komendy w module BitsTransfer.
+12.To teraz postarajmy sie uzyc tego cuda do pobrania jakiegos duzego pliku by bylo widac proces sciagania.
+*PS C:\Users\Lenovo Yofa> Start-BitsTransfer -Source https://sourceforge.net/projects/osboxes/files/v/vb/30-Lx-Lt/5.8/64bit.7z/download -Destination C:Pobrane
+Pojawilo sie pole BITS Transfer, z paskiem pobierania pliku. Konsola zostala zablokowana.
+13.Jesli chcesz to uczynic w sposob asynchroniczny (czyli taki, ktory nie blokuje konsoli) to mozna to zrobic tak.
+W ten sposob zostalo uruchomione pobieranie w sposob asynchroniczny, czyli nie blokujacy konsoli. Moge korzystac dalej z konsoli,
+PS C:\Users\Lenovo Yofa>  $job = Start-BitsTransfer -Source https://sourceforge.net/projects/osboxes/files/v/vb/30-Lx-Lt/5.8/64bit.7z/download -Destination C:Pobrane -Asynchronous -DisplayName "Moj Bits Transfer"
+plik pobiera sie w tle.
+14.By zobaczyc jaki jest stan transferu mozna uzyc takiej komendy.
+PS C:\Users\Lenovo Yofa> Get-BitsTransfer
+JobId                                DisplayName       TransferType JobState     OwnerAccount
+-----                                -----------       ------------ --------     ------------
+4a62447a-02ca-4571-ac7c-5910590310c9 Moj Bits Transfer Download     Transferring DESKTOP-B705VF8\Lenovo Yofa
+Zostala wyswietlona list pobieranych plikow, gdzie znajduje sie plik, ktory zaczalem pobierac.
```

Comment:
To wszystko się zlewa. Nie widać gdzie jest treść skryptu a gdzie Twoja odpowiedź.
Proszę wstaw jakieś interlinie tam gdzie są Twoje komentarze tak by się wyróżniały z tego całego tekstu.
To się odnosi do całego pliku a nie tylko do tego jednego miejsca.